src/components/Slider/SliderBase/SliderRail.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef } from "react";
+import PropTypes from "prop-types";
+import { bem } from "../SliderHelpers";
+
+const SliderRail = forwardRef(({ className, children, onClick }, ref) => {
+  function handleClick(e) {
+    if (typeof onClick === "function") {
```

Comment:
just a question what do you say about adding this prop as required and supply a default noop - and remove the typeof ?

Reply:
I always thought that technically it not type checking in run-time - I still can pass (as a consumer of component) not a function by mistake - and get an error.
But maybe it is ok - if consumer passing not a function - maybe we need to through error and not to take care on it silently.
Probably I'll change it ...

Reply:
Done!